src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/New.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useRef} from 'react';
+import {useTranslate} from '@akeneo-pim-community/legacy-bridge';
+
+interface NewProps {
+    createAttributeOption: (optionCode: string) => void;
+}
+
+const New = ({createAttributeOption}: NewProps) => {
+    const translate = useTranslate();
+    const newOptionCodeRef = useRef(null);
```

Comment:
```suggestion
    const newOptionCodeRef = useRef<HTMLInputElement>(null);
```
And then you can get rid of the two @ts-config below

Reply:
aaaah thanks ! I didn't understand why I had to add those @ts-config.